ui/src/main/webapp/src/app/reports/application-index/application-index.component.html

```diff
@@ -1,160 +1,213 @@
-                        </tr>
-                    </tfoot>
-                </table>
+        <div *ngIf="!reportFilter" class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
+            <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
+                <div class="card-pf-body">
+                    <div class="card-pf-top-element">
+                        <h4>Components</h4>
+                        <table class="table table-condensed">
+                            <tbody>
+                            <tr *ngFor="let statEntry of componentsStats?.entries">
+                                <td class="summaryStats">{{statEntry.name.toUpperCase()}}</td>
+                                <td><span class="highlightNumber">{{statEntry.value}}</span></td>
+                            </tr>
+                            </tbody>
+                            <tfoot>
+                            <tr class="info">
+                                <td class="summaryStats"><span class="highlightNumber">Total</span></td>
+                                <td><span class="highlightNumber">{{sumStatsList(componentsStats)}}</span></td>
+                            </tr>
+                            </tfoot>
+                        </table>
+                    </div>
+                </div>
             </div>
         </div>
 
-        <!-- Java packages usage -->
-        <div class="row col-md-12 report-index-row">
+        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12"></div>
 
-            <div class="col-md-4 chartBoundary" *ngIf="globalPackageUseData && globalPackageUseData.length > 0">
-                <h4>Java Packages Usage Chart</h4>
-                <div id="application_pie" class="windupPieGraph">
-                    <wu-package-chart
-                            [view]="[500, 150]"
-                            [results]="globalPackageUseData"
-                            [scheme]="getColorScheme(globalPackageUseData.length)"
-                    >
-                    </wu-package-chart>
+        <!-- Stats for archives, components, files -->
+        <!-- Java packages usage -->
+        <div class="col-xs-12 col-sm-6 col-md-4 col-lg-3" *ngIf="globalPackageUseData && globalPackageUseData.length > 0">
+            <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
+                <div class="card-pf-body">
+                    <h4>Java Packages Usage Chart</h4>
+                    <div id="application_pie" class="windupPieGraph">
+                        <wu-package-chart
+                                [view]="[500, 150]"
+                                [results]="globalPackageUseData"
+                                [scheme]="getColorScheme(globalPackageUseData.length)"
+                        >
+                        </wu-package-chart>
+                    </div>
+                </div>
```

Comment:
it looks like on jee-example-app packages chart overflow the card-pf-body.

Reply:
Yeah, I will file separate JIRAs for a few issues like this.